src/components/calendar.rs

```diff
@@ -622,8 +630,23 @@ impl Calendar {
     /// Provides field keys to be ignored depending on the calendar.
     pub fn field_keys_to_ignore(
         &self,
-        _keys: &[TemporalFieldKey],
+        keys: &[TemporalFieldKey],
     ) -> TemporalResult<Vec<TemporalFieldKey>> {
+        let mut ignored_keys = Vec::default();
```

Comment:
Should this vec have a minimal capacity? I think right now it will hold 3 keys before reallocating

Reply:
Oooh, potentially. I'll check that!